notebooks/syntalos_spikeinterface_pipeline.ipynb

```diff
@@ -181,9 +181,7 @@
     "    recording_processed = recording\n",
     "\n",
     "if apply_cmr:\n",
-    "    recording_processed = st.preprocessing.common_reference(recording_processed)\n",
-    "else:\n",
-    "    recording_processed = recording"
```

Comment:
Why removing the `else` here?

Reply:
If `apply_filter` is `True` and `apply_cmr` is `False`, the else would undo the filter and simply set `recording_processed` to the original recording. By that point in the logic, `recording_processed` is the only object being operated on

Reply:
sorry it's hard to have an overview of notebooks from the github page! Good to go then :)

Reply:
Yeah, agreed.

What I usually do is just look at the main code page, which _does_ render it in full notebook format rather than just code, for some reason *shrug*